### ISSUE TITLE
Plate Center not shown for APOGEE plates

### ISSUE THREAD

**Issue by bonbons0220** [sciserver/SkyServer#222]
The Plate Center is not shown for APOGEE plates, bu is shown for SDSS, SEGUE, and BOSS plates.

Steps to reproduce: 

1. Go to http://skyserver.sdss.org/dr13/en/tools/getimg/plate.aspx
2.  Select the radio button SDSS-I.
3. Leave the plate as the default.
4. Click "Get Plate" button. 
5. The plate center is shown:
```
Plate 266

Observed on MJD 51602

Plate center: RA = 145.89281, dec = 0.061309
```
6. Select the radio button APOGEE.
7. Leave on default APOGEE plate.
8. Click "Get Plate"
9. Plate Center is not shown.

**Comment by mtaghiza** [comment]
Fixed.

**Comment by bonbons0220** [comment]
Fixed in sciserver-v1.9.2.
